homeassistant/components/fritz/binary_sensor.py

```diff
@@ -17,79 +20,76 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+SENSOR_TYPES = (
```

Comment:
```suggestion
SENSOR_TYPES: tuple[BinarySensorEntityDescription, ...] = (
```
Not strictly necessary, I like to do it nevertheless.